torch/csrc/autograd/python_variable.cpp

```diff
@@ -2279,39 +2279,35 @@ void ConcretePyInterpreterVTable::python_dispatcher(
     const c10::OperatorHandle& op,
     c10::DispatchKeySet ks,
     torch::jit::Stack* stack) const {
+  py::gil_scoped_acquire g;
+  py::handle torch_api_function_overload = getTorchApiFunction(op);
+
+  c10::DispatchKey k = ks.highestPriorityTypeId();
+  auto handler = torch_api_function_overload.attr(toString(k));
```

Comment:
is handler maybe a function? does it have a .ptr attribute? i'm confused here..

Reply:
handler is a pybind11 `py::handle`, aka a PyObject.

Reply:
ok.  i guess as long as we don't expect that getattr to get invoked from multiple call sites, i'm ok with this.